shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2309,11 +2312,8 @@ \subsection{Stake Pool Registration}
 an adversarial stake pool operator could circumvent the Sybil protection
 of the pledge mechanism, by pledging stake to a pool until it attracted stake,
 and then simply pledging the stake to the next pool. The pledge will be
-enforced at the point of leader election; stake pools that do not
-honour their pledge (i.e., pools where the amount of stake delegated
-to the pool from the set of owner stake addresses is less than the pledge listed
-in its stake pool registration certificate) will be excluded from the election,
-and as a consequence forfeit their rewards for that epoch.
+enforced during the reward calculation: pools where the owners do not meet the
+pledge in a given epoch will earn no rewards for that epoch.
```

Comment:
Can we add a clarification too - as to whether only the pool owners will not receive the rewards (current case), or the delegators will suffer too , thereby damaging pool reputation.

Reply:
```suggestion
pledge in a given epoch will earn no rewards for that epoch. Note that this affects \emph{all} pool rewards, both for the operator and for pool members.
```

Reply:
Good point, it's better to spell it out. Thanks!

Reply:
the big difference and question is if the delegators will or will not see rewards when the pool owner does not meet the declared pledge. 
A wealthy attacker could register a pool with highest possible pledge (eg 10-100m) and 0% margin to attract as many delegations and stake as possible. Then break his pledge to do the same with a new pool.  
When rewards are only lost for the operator but not its delegators it's highly probably the delegated stake remain quite a while where it is. 
Well organised, this can lead to a complete market shakeout for average pool operators. In the worst case, it allows a Sybil attack

Reply:
Indeed, for Sybil protection, it's vital that delegators are hurt when the owners do not meet the pledge.